src/wallet/rpcwallet.cpp

```diff
@@ -360,11 +376,13 @@ static UniValue sendtoaddress(const JSONRPCRequest& request)
                     {"subtractfeefromamount", RPCArg::Type::BOOL, /* default */ "false", "The fee will be deducted from the amount being sent.\n"
             "                             The recipient will receive less bitcoins than you enter in the amount field."},
                     {"replaceable", RPCArg::Type::BOOL, /* default */ "wallet default", "Allow this transaction to be replaced by a transaction with higher fees via BIP 125"},
-                    {"conf_target", RPCArg::Type::NUM, /* default */ "wallet default", "Confirmation target (in blocks)"},
-                    {"estimate_mode", RPCArg::Type::STR, /* default */ "UNSET", "The fee estimate mode, must be one of:\n"
+                    {"conf_target", RPCArg::Type::NUM, /* default */ "wallet default", "Confirmation target (in blocks), or fee rate (for " + CURRENCY_UNIT + "/KB or " + CURRENCY_ATOM + "/B estimate modes)"},
```

Comment:
"KB" is 1024*1024 bytes; we should at least document it as "kB"

Reply:
Estimate modes are all uppercase, so I don't think the convention applies here.

Reply:
Since we're making them case-insensitive, it seems like a good opportunity to introduce mixed case here.

Reply:
I'm not opposed to that, though it would make the PR unnecessarily bigger.